packages/compass-components/src/components/toggle.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import LeafyGreenToggle from '@leafygreen-ui/toggle';
+
+import { Theme, useTheme } from '../hooks/use-theme';
+
+function Toggle(
+  props: React.ComponentProps<typeof LeafyGreenToggle>
```

Comment:
```suggestion
  props: Omit<React.ComponentProps<typeof LeafyGreenToggle>, 'darkMode'>
```

Reply:
Good question. I think we probably want it to be overrideable, yes, but I'm alright with either approach. If it's a component which always is displayed in a darkMode or always needs to be light mode I think we'll want to customize it and override the provider. I think we could also add more providers in that situation, but if it's just one component I think it's simpler to just override this. I'm cool with either here.

Reply:
Yeah, this makes sense! Let's keep it like that for now 👍